### PR TITLE
COCICCS-6138:T-Vault Logging enhancements - More request tracing for …

### PR DESCRIPTION
**Pull request by msivatmob**
…Safes

### CODE REVIEW

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java [hunk 1]
```diff
@@ -197,11 +197,17 @@
 	public ResponseEntity<String> createSafe(String token, Safe safe) {
 		log.debug(JSONUtil.getJSON(ImmutableMap.<String, String>builder().
 				put(LogMessage.USER, ThreadLocalContext.getCurrentMap().get(LogMessage.USER).toString()).
-				put(LogMessage.ACTION, "Create SDB").
-				put(LogMessage.MESSAGE, String.format ("Trying to Create SDB [%s]",safe.getSafeBasicDetails().getName())).
+				put(LogMessage.ACTION, TVaultConstants.CREATE_SDB).
```
Comment (alexrajan3) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java:200]: TVaultConstants class should have only common constants. Include this value inside the same class itself.

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java [hunk 2]
```diff
@@ -197,11 +197,17 @@
...
-				put(LogMessage.ACTION, "Create SDB").
-				put(LogMessage.MESSAGE, String.format ("Trying to Create SDB [%s]",safe.getSafeBasicDetails().getName())).
+				put(LogMessage.ACTION, TVaultConstants.CREATE_SDB).
+				put(LogMessage.MESSAGE, String.format ("Start trying to Create SDB [%s]",safe.getSafeBasicDetails().getName())).
 				put(LogMessage.APIURL, ThreadLocalContext.getCurrentMap().get(LogMessage.APIURL).toString()).
 				build()));
 		if (!ControllerUtil.areSDBInputsValid(safe)) {
+			log.error(JSONUtil.getJSON(ImmutableMap.<String, String>builder().
+					put(LogMessage.USER, ThreadLocalContext.getCurrentMap().get(LogMessage.USER).toString()).
+					put(LogMessage.ACTION, TVaultConstants.CREATE_SDB).
```
Comment (alexrajan3) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java:207]: Remove this from TVaultConstants

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java [hunk 3]
```diff
@@ -212,18 +218,48 @@
 		basicDetails.setAppName(applicationName);
 		safe.setSafeBasicDetails(basicDetails);
 		if(applicationName == null) {
+			log.error(JSONUtil.getJSON(ImmutableMap.<String, String>builder().
+					put(LogMessage.USER, ThreadLocalContext.getCurrentMap().get(LogMessage.USER).toString()).
+					put(LogMessage.ACTION, TVaultConstants.CREATE_SDB).
```
Comment (alexrajan3) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java:223]: Remove this from TVaultConstants

**Review by alexrajan3 — CHANGES_REQUESTED** [review]

**Review by sarathmohan4 — DISMISSED** [review]

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java (hunk 1 above)
Comment (msivatmob) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java]: Constant values are included in same class itself.

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java (hunk 3 above)
Comment (msivatmob) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java]: Removed.

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java (hunk 2 above)
Comment (msivatmob) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java]: Removed.

**Pull request merged**